examples/jsm/utils/WebGPUTextureUtils.js

```diff
@@ -19,6 +19,13 @@ export async function decompress( blitTexture, maxTextureSize = Infinity, render
 	}
 
 	const material = new NodeMaterial();
+
+	// disable uv transform
+	const currentTextureMatrix = blitTexture.matrix.clone();
+	const currentTextureMatrixAutoUpdate = blitTexture.matrixAutoUpdate;
+	blitTexture.matrix.identity();
+	blitTexture.matrixAutoUpdate = false;
+
 	material.fragmentNode = texture( blitTexture ).uv( uv().flipY() );
```

Comment:
@Mugen87 If we use `uv` in the `texture()` it should be ignored, `matrix` is only used by default.

```js
material.fragmentNode = texture( blitTexture, uv().flipY() )
```

Reply:
Nice! This greatly simplifies the code. I'll make a new PR.